test.js

```diff
@@ -1,22 +1,16 @@
-'use strict';
-var childProcess = require('child_process');
-var test = require('ava');
-var taskkill = require('./');
+import childProcess from 'child_process';
+import test from 'ava';
+import fn from './';
```

Comment:
@SamVerschueren A module is just a file containing exports. The main export can be any type. I thought `m` was nice because it could mean `module` or `main`. I'm totally open for suggestions?